build/security/resources/contrast/contrast_security.yaml

```diff
@@ -0,0 +1,9 @@
+api: 
+  url: https://app.contrastsecurity.com/Contrast
+  api_key: API_KEY
+  service_key: SERVICE_KEY
+  user_name: AGENT_NAME
+application: 
+  metadata: bU="BUSINESS_UNIT"
+  name: "IBM FHIR Server"
+  version: "99-SNAPSHOT"
```

Comment:
what is this the version of?

Reply:
it's actually for contrast, I just needed something not bound to 4.X (I didn't want to associate it with a release)